### PR TITLE
Release 0.7.1 part 1/3: CHANGELOG, README, Makefile

### PR DESCRIPTION
**Pull request by khoang98**
**Is this a bug fix or adding new feature?**
Neither

**What is this PR about? / Why do we need it?**
Need to trigger a new build of the driver to update packages.

**What testing is done?** 
Nothing significant added.

### CODE REVIEW

File: CHANGELOG-0.x.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# v0.7.1
+
+### New features
+
+### Misc.
+* Release 0.7.0 part 3/3 ([#224](https://github.com/kubernetes-sigs/aws-fsx-csi-driver/pull/224), [@khoang98](https://github.com/khoang98))
```
Comment (wongma7) [inline CHANGELOG-0.x.md:6]: probably we can remove this and just say 'updating to latest al2 base image ' or something like that

**Comment by wongma7** [comment]
thanks.

the diff looks okay but git history weird.

Try doing `git rebase -i` and just dropping every commit but the last. alternatively , can open a new PR as well to avoid the master-release branch untangling

**Comment by wongma7** [comment]
after this merges I will do a build and then can merge part 2/3

File: CHANGELOG-0.x.md (hunk 1 above)
Comment (khoang98) [reply CHANGELOG-0.x.md:6]: Sounds good - going to close this PR and make this change in a different one.

**Pull request closed without merging**